android/src/main/java/com/github/alinz/reactnativewebviewbridge/WebViewModule.java

```diff
@@ -0,0 +1,77 @@
+package com.github.alinz.reactnativewebviewbridge;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ActivityEventListener;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.ClipData;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.webkit.ValueCallback;
+import android.util.Log;
+
+import com.facebook.react.common.annotations.VisibleForTesting;
+
+public class WebViewModule extends ReactContextBaseJavaModule implements ActivityEventListener {
+    public ValueCallback<Uri> mUploadMessage;
+    public ValueCallback<Uri[]> mUploadCallbackAboveL;
```

Comment:
should be private
should not have `m` prefix
and we don't support even Lolipop, so `AboveL` seems pointless